.github/workflows/image-publish.yaml

```diff
@@ -0,0 +1,139 @@
+name: Docker build and push
+
+on:
+  release:
+    types: [published, edited]
+
+env:
+  dockerhub_user: kloeckneri
+  manufacturer: kloeckner-i
+  product_name: db-auth-gateway
+  main_go_path: ./cmd
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - { go: "1.17", goos: "linux", goarch: "amd64", dockerArch: "amd64" }
```

Comment:
go.mod is with golang 1.18

Reply:
The go version in the workflow is updated to 1.18